src/new-client/src/App/Footer/StatusButton/StatusButton.tsx

```diff
@@ -85,7 +86,7 @@ export const StatusButton: React.FC = () => {
   const url = "https://web-demo.adaptivecluster.com"
   const ref = useRef<HTMLDivElement>(null)
   const { displayMenu, setDisplayMenu } = usePopUpMenu(ref)
-  const services: ServiceStatus[] = []
+  const services: ServiceStatus[] = useServiceStatus()
```

Comment:
I would much rather to have a hook defined just before the `StatusButton` component like: `useApplicationStatus`.